flow-typed/types/external-request-loader.js

```diff
@@ -0,0 +1,5 @@
+// @flow
+declare type ExternalRequestLoader = {
```

Comment:
Not sure we need to call this `external`, simply request loader, as this is correct for both internal and external

Reply:
done